runtime/oti/j9nonbuilder.h

```diff
@@ -3569,6 +3569,17 @@ typedef struct J9HookedNative {
 } J9HookedNative;
 
 /* @ddr_namespace: map_to_type=J9ClassLoader */
+typedef struct J9RAMClassFreeLists {
+	struct J9RAMClassFreeListBlock* ramClassTinyBlockFreeList;
+	struct J9RAMClassFreeListBlock* ramClassSmallBlockFreeList;
+	struct J9RAMClassFreeListBlock* ramClassLargeBlockFreeList;
+} J9RAMClassFreeLists;
+
+typedef struct J9RAMClassUDATABlockFreeList {
```

Comment:
I would suggest adding a `UDATA *` field to J9RAMClassFreeLists instead of having a separate structure to hold these values.